strawberry/experimental/pydantic/fields.py

```diff
@@ -16,10 +16,6 @@
     "StrictStr": str,
     "ConstrainedBytes": bytes,
     "conbytes": bytes,
-    "ConstrainedList": None,
-    "conlist": None,
-    "ConstrainedSet": None,
-    "conset": None,
```

Comment:
i still removed CosntrainedSet because it doesn't work to throw UnsupportedTypeError in the first place ;)
also we previously discussed we removing the blacklist.

which i'll do in another MR https://discord.com/channels/689806334337482765/938317339097698354/938368333860982794

Reply:
can you open an issue so we keep track of it? 😊

Reply:
@patrick91 filed a quick one here, i'll elaborate another time :D https://github.com/strawberry-graphql/strawberry/issues/1670